{
  "comments": [
    {
      "key": {
        "uuid": "b0718a1d_1f961b4b",
        "filename": "init/init.cpp",
        "patchSetId": 3
      },
      "lineNbr": 964,
      "author": {
        "id": 1010111
      },
      "writtenOn": "2015-04-23T20:05:02Z",
      "side": 1,
      "message": "Doesn\u0027t appear that most ERROR() messages include the function name, but if you are going to do to that, I\u0027d use __func__.  Or if you prefer this, there is an extraneous d above.",
      "revId": "2db66e2a1f45c153aa8e6856374acbff5e2bdd8a",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    }
  ]
}